server/server.js

```diff
@@ -38,68 +38,90 @@ app.use(braveAlerter.getRouter())
 async function handleValidRequest(button, numPresses, batteryLevel) {
     helpers.log('UUID: ' + button.button_id.toString() + ' SerialNumber: ' + button.button_serial_number + ' Unit: ' + button.unit.toString() + ' Presses: ' + numPresses.toString() + ' BatteryLevel: ' + batteryLevel)
 
-    let client = await db.beginTransaction()
+    let client 
+    
+    try {
+        client = await db.beginTransaction()
+        if (client === null) {
+            helpers.log(`alertSessionChangedCallback: Error starting transaction`)
```

Comment:
I think this log message needs to be updated?

Reply:
Fixed. Thanks!